Agilefreaks/FreaksCatalog-API
# [119] Add level to a freak

## Description
In this PR we can create a freak with a level

## Related
[[42] Add Level to a Freak](https://trello.com/c/XsZjZ9EV/119-add-level-to-a-freak)

## Steps to Test or Reproduce
Go in Insomnia and create a freak. You can use this graphql mutation:
```
mutation($norm_id: ID!, $role_id: ID!, $level_id: ID!) {
	freakCreate(
		firstName: "john",
		lastName: "doe",
		description: "ceva",
		email: "freak@gmail.com",
		normId: $norm_id,
		roleId: $role_id,
		levelId: $level_id
	) {
		firstName
		lastName
		description
		email
        norm {
            name
        }
		role {
			name
		}
		level{
			name
		}
	}
}

```
Query freaks to see that the new freak was created: 

```
{
  freaks {
    nodes {
      level {
        id
        name
      }
    }
  }
}
```
